pkg/util/validation/limits.go

```diff
@@ -443,6 +446,14 @@ func (o *Overrides) IngestionBurstSize(userID string) int {
 	return o.getOverridesForUser(userID).IngestionBurstSize
 }
 
+func (o *Overrides) IngestionBurstFactor(userID string) float64 {
+	burstFactor := o.getOverridesForUser(userID).IngestionBurstFactor
+	if burstFactor < 1 {
```

Comment:
I'm not a big fan of silently setting this to 0 if the input is less than 1. Based on the other configs this doesn't seem too far off what we do in other places but I'm not sure if there's a better way.

Reply:
I don't feel strong about this, but I think what you're doing is fine. I don't see many alternatives.